source/Calamari.Common/Variables/SpecialVariables.cs

```diff
@@ -0,0 +1,26 @@
+namespace Calamari.Common.Variables
+{
+    public static class SpecialVariables
+    {
+        
+        public static readonly string CalamariWorkingDirectory = "OctopusCalamariWorkingDirectory";
+        public static readonly string RetentionPolicySet = "OctopusRetentionPolicySet";
+        public static readonly string PrintVariables = "OctopusPrintVariables";
+        public static readonly string PrintEvaluatedVariables = "OctopusPrintEvaluatedVariables";
+        
+        // Everything below here needs a review
+        public static readonly string CopyWorkingDirectoryIncludingKeyTo = "Octopus.Calamari.CopyWorkingDirectoryIncludingKeyTo";
+        public static readonly string SkipFreeDiskSpaceCheck = "OctopusSkipFreeDiskSpaceCheck";
+        public static readonly string FreeDiskSpaceOverrideInMegaBytes = "OctopusFreeDiskSpaceOverrideInMegaBytes";
+        public const string AdditionalVariablesPath = "Octopus.AdditionalVariablesPath";
+        public const string OriginalPackageDirectoryPath = "OctopusOriginalPackageDirectoryPath";
+        public const string LastErrorMessage = "OctopusLastErrorMessage";
```

Comment:
In my draft PR I inlined it as it's just used in one place https://github.com/OctopusDeploy/Calamari/pull/539/files#diff-725458750185f37d17668ba66cd4fc6aR70